beetsplug/chroma.py

```diff
@@ -88,16 +113,24 @@ def acoustid_match(log, path):
         return None
     _acoustids[path] = result['id']
 
-    # Get recording and releases from the result.
+    # Get recording and releases from the result,
+    #  sorted by date and/or country depending on 'match'>'preferred' config.
```

Comment:
We usually name config options using dot syntax. For example, "the match.preferred configuration option" here. Also, looks like this comment needs to get rewrapped.